packages/@aws-cdk/aws-lambda-python/lib/function.ts

```diff
@@ -34,6 +34,18 @@ export interface PythonFunctionProps extends lambda.FunctionOptions {
    * @default lambda.Runtime.PYTHON_3_7
    */
   readonly runtime?: lambda.Runtime;
+
+  /**
+   * Install dependencies
+   * @default true
+   */
+  readonly installDependencies?: boolean;
+
+  /**
+   * Use a dependency lambda layer for packages.
+   * @default false
+   */
+  readonly useDependencyLayer?: boolean;
```

Comment:
Would it be clearer (for the consumer) if we used an enum in place of these two booleans? With options like `NONE`, `INSTALL_INLINE`, `INSTALL_TO_LAYER`?

Reply:
I like your idea.

Reply:
I've pushed up a change which is very similar to this suggestion.